.github/workflows/release.yml

```diff
@@ -0,0 +1,22 @@
+name: Release
+on:
+  push:
+    branches:
+      - master
```

Comment:
Should these include the release branches?  OR are you thinking, since there's no npm here, cert and hotfixes would just be done with a github release on any branch anyways?

Reply:
Yeah I've been leaving this off the non-NPM stuff to simplify them.